index.html

```diff
@@ -453,14 +453,12 @@ <h3>Claims</h3>
       <h2>Credentials</h2>
 
       <p>
-A <a>credential</a> is set of one or more <a>claims</a> about a <a>subject</a>.
-It typically includes an identifier to uniquely identify the
-credential. Credential metadata may also be included to express concepts
-such as when the credential expires. A digital signature is almost always
-appended by the <a>issuer</a> of the credential to enable verifiability of
-the credential. Therefore, a <strong>verifiable credential</strong> is a set
-of claims that are tamper-resistant and whose authorship can be
-cryptographically verified.
+A <a>credential</a> is set of one or more <a>claims</a> asserted by the issuer.
+It includes an identifier to uniquely identify the credential, as well as
```

Comment:
It does not always include the identifier to uniquely identify the credential.

Reply:
I am sure this is news to many people in the working group :-)

Reply:
Which just goes to show why ambiguity needs to be removed from the specification!

Reply:
One way to resolve this is to just change instances like this to use `SHOULD` language and leave it at that:

`It SHOULD include an identifier to uniquely identify the credential...`